boards/common/samd21-arduino-bootloader/Makefile.include

```diff
@@ -0,0 +1,33 @@
+# setup the flash tool used
+# Bossa is the default programmer
+PROGRAMMER ?= bossa
+
+ifeq ($(PROGRAMMER),bossa)
+  # by default, we use BOSSA to flash this board and take into account the
+  # preinstalled Arduino bootloader. ROM_OFFSET skips the space taken by
+  # such bootloader.
+  ROM_OFFSET ?= 0x2000
+  BOSSA_ARDUINO_PREFLASH = yes
+  PREFLASH_DELAY = 1
+
+  ifneq (,$(filter reset flash flash-only, $(MAKECMDGOALS)))
+    # Add 2 seconds delay before opening terminal: this is required when opening
+    # the terminal right after flashing. In this case, the stdio over USB needs
+    # some time after reset before being ready.
+    TERM_DELAY = 2
```

Comment:
Does it make sense to have this configurable for the end user? As in:
```Suggestion
    TERM_DELAY ?= 2
```

Reply:
Fixed